style.css

```diff
@@ -381,13 +367,19 @@ hr.solid {
   padding: 114px 24px;
 }
 
-h2.about-title {
+.about-title {
```

Comment:
- [Optional] Consider adjusting the left and right margins of the `about-child` element so that it will match the design template. You are expected to stick to design as much as possible. Try changing the margins to;
```css
.about-child {
    margin: auto 1.5rem;
    padding: 114px 24px;
}
```
### Your design
![f1](https://user-images.githubusercontent.com/94551484/198851511-1ced6160-71df-4412-b74b-838480927e0f.png)

### Design template
![1](https://user-images.githubusercontent.com/94551484/198851516-9dd7c808-892d-4de5-838f-02ea96184d65.png)

Reply:
THANKS! you are really awesome,
will do my best to implement the optional changes.